src/psyclone/tests/psyir/transformations/transformations_test.py

```diff
@@ -222,6 +222,34 @@ def test_ompsingle_nested():
            " be enclosed by a OMPSingleTrans transformation"
            in str(err.value))
 
+
+# Tests for OMPMasterTrans
+def test_ompmaster():
+    ''' Generic tests for the OMPMasterTrans transformation class '''
+    trans = OMPMasterTrans()
+    assert trans.name == "OMPMasterTrans"
+    assert str(trans) == "Insert an OpenMP Master region"
+
+
+def test_ompmaster_nested():
+    '''Tests to check OMPMasterTrans rejects being applied to another
+    OMPMasterTrans'''
+
+    _, invoke_info = parse(os.path.join(GOCEAN_BASE_PATH, "single_invoke.f90"),
+                           api="gocean1.0")
+    master = OMPMasterTrans()
+    psy = PSyFactory("gocean1.0", distributed_memory=False).\
+        create(invoke_info)
+    schedule = psy.invokes.invoke_list[0].schedule
+
+    master.apply(schedule[0])
```

Comment:
Maybe expand this line to something like:
```
 # Successful transformation
 node = schedule[0]
 master.apply(node)
 assert isinstance(schedule[0], OMPMasterDirective)
 assert schedule[0].dir_body[0] is node
```
to check the successful case.

Reply:
Done